apps/backend/src/users/user.entity.ts

```diff
@@ -18,7 +19,7 @@ export class User {
   email: string;
 
   @Column()
-  profilePicture: string;
+  profilePicture: string | null;
```

Comment:
💭 maybe instead of having a null profilePicture, we could have a placeholder picture? Just an idea, maybe for a future enhancement